src/types.tsx

```diff
@@ -11,7 +11,9 @@ export interface FormikValues {
  * Should be always be and object of strings, but any is allowed to support i18n libraries.
```

Comment:
This documentation line has problems, but it suggests that the final error type should be `any`.

Reply:
I see. so I'll change `string` to `string | any` 👌

Reply:
Why not just `any`? 😃

Reply:
just any OK👌I referred to previous commit. but it was old 😅

Reply:
I fixed errors type and merged commit message.